en/brands.ftl

```diff
@@ -13,7 +13,6 @@
 -brand-name-facebook = Facebook
 -brand-name-github = GitHub
 -brand-name-google = Google
--brand-name-hulu = Hulu
```

Comment:
Why is this removed? The brand term was added in the adblocker.ftl file

Reply:
It wasn't in my PR. Investigating...

Reply:
Oh I see, the file in this repo is ahead of the one in bedrock so the robot thought this line was deleted (since it's not in bedrock yet). Hulu is being added in https://github.com/mozilla/bedrock/pull/9439/ that hasn't merged yet, but I'll review that now and it will generate a new PR when it merges.

Reply:
So... that PR had `[skip l10n]` so I don't think the robot will make an automated PR to replace this one. I think now the only way to fix this is to manually open a replacement PR, so I'll do that.

Reply:
Oh hang on, I can push to this branch. (sorry for all the comments, just talking to myself...)

Reply:
Really appreciate you taking the time to get  to the bottom of this.